src/main/groovy/edu/oregonstate/mist/locations/LocationUtil.groovy

```diff
@@ -33,87 +18,4 @@ class LocationUtil {
     public static String getMD5Hash(String content) {
         MessageDigest.getInstance("MD5").digest(content.bytes).encodeHex().toString()
```

Comment:
This class just contains the md5hash function now. Is there a different class we should move this function to? This class doesn't even use the `locationConfiguration` map anymore.

Reply:
I'd be happy to. Suggestions? It doesn't really seem to belong anywhere. Its only remaining use seems to be by the BaseType class, to compute location ids. I could move it to a new Util class, but it's already in a Util class, so the benefit seems small.

Reply:
Fair enough. I don't have any suggestions, so I guess it can stay here for now.